cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/GraphCommand.java

```diff
@@ -0,0 +1,193 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.cli.cmd;
+
+import io.ballerina.cli.BLauncherCmd;
+import io.ballerina.cli.TaskExecutor;
+import io.ballerina.cli.task.CleanTargetDirTask;
+import io.ballerina.cli.task.CreateDependencyGraphTask;
+import io.ballerina.cli.task.ResolveMavenDependenciesTask;
+import io.ballerina.cli.utils.FileUtils;
+import io.ballerina.cli.utils.ProjectUtils;
+import io.ballerina.projects.BuildOptions;
+import io.ballerina.projects.Project;
+import io.ballerina.projects.ProjectException;
+import io.ballerina.projects.directory.BuildProject;
+import io.ballerina.projects.directory.SingleFileProject;
+import io.ballerina.projects.util.ProjectConstants;
+import org.ballerinalang.toml.exceptions.SettingsTomlException;
+import org.wso2.ballerinalang.util.RepoUtils;
+import picocli.CommandLine;
+
+import java.io.PrintStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static io.ballerina.cli.cmd.Constants.GRAPH_COMMAND;
+
+/**
+ * This class represents the "bal graph" command.
+ *
+ * @since 2201.2.0
+ */
+@CommandLine.Command(name = GRAPH_COMMAND, description = "bal graph - Print the dependency graph")
+public class GraphCommand implements BLauncherCmd {
+    private final PrintStream outStream;
+    private final PrintStream errStream;
+    private final boolean exitWhenFinish;
+    @CommandLine.Parameters(arity = "0..1")
```

Comment:
Let's separate the parameters with a new line for clarity. Please address other parameters as well
```suggestion
    
    @CommandLine.Parameters(arity = "0..1")
```